docs/src/content/docs/es/guides/components.mdx

```diff
@@ -199,3 +199,49 @@ El código anterior genera lo siguiente en la página:
 A continuación se muestra una lista de todos los iconos disponibles con sus nombres asociados. Haz clic en un ícono para copiar el código del componente correspondiente.
 
 <IconsList />
+
+### Code
```

Comment:
I was thinking about translating it, but since the Component is called like that and we're not supposed to be translating the name of that component cause it could be misleading. That's why I left it as is.

Reply:
In my opinion we could translate this title without problems, just like the title of the `Tabs` component is translated as `Pestañas` or the `Icon` component to `Iconos`.